src/ResourceManager/Websites/Commands.Websites/Cmdlets/Certificates/GetAzureWebAppCertificate.cs

```diff
@@ -19,7 +19,7 @@ namespace Microsoft.Azure.Commands.WebApps.Cmdlets.WebApps
     /// this commandlet will let you get existing web app certificates using ARM APIs
     /// </summary>
     [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "WebAppCertificate")]
-    [OutputType(typeof(Certificate))]
+    [OutputType(typeof(PSCertificate))]
```

Comment:
Looks like you didn't change the output below - unless CmdletHelpers.GetCertificates now returns a PSCertificate

Reply:
Done.